Features/Perpetuals/Sources/ViewModels/PerpetualSceneViewModel.swift

```diff
@@ -224,7 +224,7 @@ public extension PerpetualSceneViewModel {
         )
     }
 
-    private func createTransferData(direction: PerpetualDirection) -> PerpetualTransferData? {
+    private func createTransferData(direction: PerpetualDirection, leverage: UInt8? = nil) -> PerpetualTransferData? {
```

Comment:
```suggestion
    private func createTransferData(direction: PerpetualDirection, leverage: UInt8) -> PerpetualTransferData? {
```

Handle leverage on higher level, because in the future we will pull leverage from preferences. 

`createTransferData` should not hide business logic details